src/other/Table.js

```diff
@@ -125,11 +164,11 @@
             })
         ;
         th.select(".thIcon")
-            .text(function (column) {
+            .text(function (column, d) {
```

Comment:
This might be confusing for some as we tend to use "d, idx" or "d, i" - this should be "column, idx"?